### PR TITLE
announcement: camunda/zeebe#16406 announcement

### PR DESCRIPTION
**Pull request by megglos**
## Description

Announcement on a blocking bug in recent 8.1 and 8.2 patches.

We need to release this asap.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: versioned_docs/version-8.1/reference/announcements.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ title: "Announcements"
...
 ---
 
+## Camunda 8.1
+
+Release date: 11th of October 2022
+
+End of maintenance: 10th of April 2024
+
+[Release notes](https://github.com/camunda/camunda-platform/releases/tag/8.1.0)
+[Release blog](https://camunda.com/blog/2022/10/camunda-platform-8-1-released-whats-new/)
```
Comment (korthout) [inline versioned_docs/version-8.1/reference/announcements.md:14]: 😍

File: versioned_docs/version-8.1/reference/announcements.md [hunk 2]
```diff
@@ -4,6 +4,21 @@ title: "Announcements"
...
+
```
Comment (megglos) [inline versioned_docs/version-8.1/reference/announcements.md:15]: the 8.1 section was missing on 8.1 😅

**Review by korthout — APPROVED** [review]
Thanks @megglos 💯 

LGTM 👍

**Comment by megglos** [comment]
@christinaausley another announcement for an incident, could you take that asap and release after? Thank you, thank you, thank you 😅

**Review by christinaausley — APPROVED** [review]

**Pull request merged**